go.mod

```diff
@@ -3,15 +3,15 @@ module github.com/blinklabs-io/tx-submit-api
 go 1.19
 
 require (
-	github.com/blinklabs-io/gouroboros v0.40.1
+	github.com/blinklabs-io/gouroboros v0.41.0
 	github.com/fxamacker/cbor/v2 v2.4.0
 	github.com/gin-contrib/zap v0.1.0
 	github.com/gin-gonic/gin v1.9.0
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/penglongli/gin-metrics v0.1.10
 	github.com/swaggo/files v1.0.1
 	github.com/swaggo/gin-swagger v1.6.0
-	github.com/swaggo/swag v1.16.1
+	github.com/swaggo/swag v1.8.12
```

Comment:
Why downgrade this?

Reply:
I can't find that 1.16.1 on the swag GitHub, so I was unable to upgrade locally.